pages/1.13/security/oss/index.md

```diff
@@ -8,15 +8,33 @@ menuWeight: 80
 
 Ensure the network is setup according to the information for [securing your cluster](/1.13/administering-clusters/securing-your-cluster/).
 
-You can enable authentication in your datacenter with DC/OS [oauth](https://github.com/dcos/dcos-oauth). Authentication is managed through the DC/OS web interface. The Admin Router enforces access control.
-
-Out-of-the-box DC/OS has an OpenID Connect 1.0 endpoint at [dcos.auth0.com](https://dcos.auth0.com/.well-known/openid-configuration) (in cooperation with [Auth0](https://auth0.com/)) with connections to Google, GitHub, and Microsoft to provide basic authentication for DC/OS installations. DC/OS automatically adds the first user that logs in to the DC/OS cluster.
+Authentication in DC/OS is implemented via dedicated a DC/OS Authentication token.
```

Comment:
This sentence is a bit broken.

If you write "Authentication in DC/OS is implemented via a dedicated DC/OS Authentication token" I am not sure if this has value to the reader. What do we want to say, really?

Reply:
Removed this sentence. What it should express is that DC/OS uses tokens for authentication.